firestore/integration_test_internal/src/firestore_test.cc

```diff
@@ -1348,6 +1362,43 @@ TEST_F(FirestoreIntegrationTest, RestartFirestoreLeadsToNewInstance) {
   delete db1;
 }
 
+TEST_F(FirestoreIntegrationTest,
+       RestartFirestoreLeadsToNewNamedDatabaseInstance) {
+  // TODO(Mila): Run this test against emulator.
+  GTEST_SKIP();
```

Comment:
skipping the test for now, will run this againt emulator once configuration is done. 
I am only able to find `SKIP_TEST_ON_EMULATOR `,`SKIP_TEST_ON_SIMULATOR`, but no flag like `SKIP_TEST_ON_NOT_EMULATOR`. How do we skip tests against prod? 🤔

Reply:
Looking at the code, these constants refer to the mobile/iPhone/device emulator, not the Firebase Emulator.

Reply:
So, that means we need to create a new flag to skip or test-only on firestore emulators? Or, we need to wait till the multi-db is supported in production database to be able to run the integration tests. 🤔

Reply:
created a new method `firestore::IsUsingFirestoreEmulator()` to skip the test case if it is not running against firestore emulator